### PR TITLE
Chinese translation

### PR DESCRIPTION
**Pull request by squirrelsc**

### CODE REVIEW

**Comment by PurityFan** [comment]
I added BOHB's usage translation in builtin-Tuner.md, can you help add that paragraph?

**Comment by squirrelsc** [comment]
> I added BOHB's usage translation in builtin-Tuner.md, can you help add that paragraph?

Added

File: README_zh_CN.md [hunk 1]
```diff
@@ -69,7 +70,7 @@ NNI (Neural Network Intelligence) 是自动机器学习（AutoML）的工具包
       </td>
       <td>
       <ul>
-        <li><a href="docs/zh_CN/tutorial_1_CR_exp_local_api.md">本地计算机</a></li>
+        <li><a href="docs/en_US/LocalMode.md">本地计算机</a></li>
```
Comment (scarlett2018) [inline README_zh_CN.md]: should point to zh_CN?

File: README_zh_CN.md [hunk 2]
```diff
@@ -60,7 +60,8 @@ NNI (Neural Network Intelligence) 是自动机器学习（AutoML）的工具包
           <li><a href="docs/zh_CN/Builtin_Tuner.md#NetworkMorphism">Network Morphism</a></li>
           <li><a href="examples/tuners/enas_nni/README_zh_CN.md">ENAS</a></li>
           <li><a href="docs/zh_CN/Builtin_Tuner.md#NetworkMorphism#MetisTuner">Metis Tuner</a></li>
-        </ul> 
+          <li><a href="docs/en_US/Builtin_Tuner.md#BOHB">BOHB</a></li>
```
Comment (scarlett2018) [inline README_zh_CN.md]: should point to zh_CN?

File: README_zh_CN.md [hunk 3]
```diff
@@ -193,7 +194,7 @@ NNI (Neural Network Intelligence) 是自动机器学习（AutoML）的工具包
 
 ## **教程**
 
-* [在本机运行 Experiment (支持多 GPU 卡)](docs/zh_CN/tutorial_1_CR_exp_local_api.md)
+* [在本机运行 Experiment (支持多 GPU 卡)](docs/en_US/LocalMode.md)
```
Comment (scarlett2018) [inline README_zh_CN.md]: should point to zh_CN?

Comment (squirrelsc) [reply README_zh_CN.md]: fixed

File: README_zh_CN.md (hunk 2 above)
Comment (squirrelsc) [reply README_zh_CN.md]: fixed

File: README_zh_CN.md (hunk 1 above)
Comment (squirrelsc) [reply README_zh_CN.md]: fixed

**Pull request merged**